minitwit_frontend/minitwit_frontend/settings.py

```diff
@@ -20,13 +20,16 @@
 # See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.getenv('MINITWIT_SECRET_KEY') or 'vk61e&%q!ggpvhg=nljgy+vg8-tq+5#z%4pesej(wf0^vz%7au'
+SECRET_KEY = os.getenv('SECRET_KEY','vk61e&%q!ggpvhg=nljgy+vg8-tq+5#z%4pesej(wf0^vz%7au')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = bool(os.getenv('DEBUG', 0))
-# DEBUG = bool(int(os.environ.get('DEBUG', 0)))
+DEBUG = bool(int(os.getenv('DEBUG', 1)))
```

Comment:
This should default to 0

Reply:
This has been corrected. The file is outdated now.